authapi/models.py

```diff
@@ -1 +1,19 @@
-# Create your models here.
+from django.contrib.auth.models import User
+from django.db import models
+
+
+class SeedOrganization(models.Model):
+    # TODO: Implement SeedOrganization model
+    users = models.ManyToManyField(User)
+
+
+class SeedPermission(models.Model):
+    # TODO: Implement SeedPermission model
+    pass
+
+
+class SeedTeam(models.Model):
+    # TODO: Implement SeedTeam model
+    organization = models.ForeignKey(SeedOrganization)
+    users = models.ManyToManyField(User)
+    permissions = models.ManyToManyField(SeedPermission)
```

Comment:
So I tried it, but the issue is that the `Group` model has a `permissions` field, which is a many-to-many link to the `Permission` model. The issue with that is that we want our own custom permission model to be linked to our `Team` model, and we can't override the `permissions` field, so we would have to add another field, and I couldn't really think of a good way of doing this, so I decided to create custom `SeedTeam` and `SeedPermission` models, instead of extending the `Group` and `Permission` models.

Any thoughts of a better way of doing it? The only thing that using `Group` gives us is a function on the `User` model to get all permissions assigned to the user through that user's groups.